### DISCUSSION TITLE
Give us the option to load emotes with the 'Get User Emotes' endpoint

### DISCUSSION THREAD

**Discussion by occluder** [Chatterino/chatterino2#5531] [Ideas]
From what I understand, the reason why this endpoint is not currently used is because it's slow and requires multiple calls depending on how many subscriptions users have. However, I would still like to use it because it loads my personal emotes as well. 

From my testing, it would take 4 calls to load all my emotes, the speed of which is something I am (and probably many others would be) comfortable with. Thus, I want to have the option to use it.

**Comment by Nerixyz** [comment]
Rebased https://github.com/Chatterino/chatterino2/pull/5239; I'll make some adjustments to make it a bit more usable while emotes are loading and to save some memory.
